sway/commands/output.c

```diff
@@ -36,7 +36,32 @@ struct cmd_results *cmd_output(int argc, char **argv) {
 				"Refusing to configure the no op output");
 	}
 
-	struct output_config *output = new_output_config(argv[0]);
+	struct output_config *output = NULL;
+	if (strcmp(argv[0], "-") == 0 || strcmp(argv[0], "--") == 0) {
```

Comment:
The right side of the `||` will always be true when the left side is true and can be omitted

Reply:
strcmp will take the null terminator into account, so it won't. That would be true with strncmp.

Reply:
derp